.travis.yml

```diff
@@ -26,6 +23,9 @@ script:
     //particles/...
     //src/...
     //src_kt/...
+  - npm run test-with-start
```

Comment:
Could be worth doing this move in a separate tiny PR to land first in case it causes some issue, to observe separately, but up to you.

Reply:
Yes, wise ... see #3890